edx_proctoring/templates/proctored_exam/confirm-decline.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="msg-content">
       {% blocktrans %}
-        If you take this exam without proctoring, you will <strong> no longer be eligible for academic credit. </strong>
+        If you take this exam without proctoring, you will not be eligible for course credit or the MicroMasters credential if either applies to this course.
```

Comment:
For now we can leave this specific reference, as we're only relying on this for the 1 case. That being said, we know we need to go back and update all this to be more generalized, which follow-on stories can address. Ok for now though IMO.